src/components/home/Form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../scss/Form.scss';
+
+const Form = ({ onChange }) => (
+  <div className="form-container d-flex">
+    <form>
+      <input type="text" placeholder="Search Country" onChange={onChange} />
+    </form>
+  </div>
```

Comment:
- [ ] [OPTIONAL] when using the search option all lowercase it doesn't find the countries, It might be nice adding this functionality.